pynest/nest/tests/test_sp/test_all.py

```diff
@@ -28,12 +28,38 @@
 from . import test_sp_manager
 from . import test_disconnect
 from . import test_disconnect_multiple
-from . import test_issue_578_sp
+from . import test_enable_multithread
+try:
+    from mpi4py import MPI
+except ImportError:
+    # Test without MPI
+    print ("MPI is not available. Skipping MPI tests.")
+    mpi_test = 0
+else:
+    # Test with MPI
+    mpi_test = 1
+    print ("Testing with MPI")
+    from subprocess import call
+    import sys
+    import os
 
 __author__ = 'naveau'
 
 
 def suite():
+    # MPI tests
+    if mpi_test:
+        try:
+            command = getMPITestCommand()
+            command = command.replace(
+                "  scriptfile", "test_sp/mpitest_issue_578_sp.py")
+            print ("Executing test with command: " + command)
```

Comment:
Given the changes to the `getMPITestCommand` function detailed below, you can replace lines 53-55 by just
```
command = getMPITestCommand(2, "test_sp/mpitest_issue_578_sp.py")
```